Editor/Window/AwsUserProfilesPage.cs

```diff
@@ -135,12 +134,13 @@ private void SetupButtonCallbacks()
             });
             _container.Q<Button>("UserProfilePageBootstrapStartButton").RegisterCallback<ClickEvent>(_ =>
             {
-                BootstrapSettings.RefreshBucketName();
-                OpenS3Popup(_stateManager.BucketName);
+                _bootstrapSettings.RefreshBucketName();
+                OpenS3Popup(_bootstrapSettings.BucketName);
```

Comment:
What's the difference? Isn't _stateManager required to be consistent with the current user profile being used? I would expect it to contain the right BucketName

Reply:
At this point in the flow they havnt set a bucket name, and BootstrapSettings will generate an example bucket name for them to use